.changesets/632-sentinel-env-vars.md

```
@@ -0,0 +1,41 @@
+- `$fake_uuid_key`
+- `$fake_base64_key`
+- `$fake_url_base64_key`
+- `$fake_hex_key`
+- `$fake_email`
+
+**New jaq helpers** usable in both `--env` and `--hook` scripts:
+- `bearer(token)` — returns `"Bearer <token>"`
+- `basic(user; pass)` — returns `"Basic <base64(user:pass)>"`
+
+Example — GitHub token injection:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)